core/chains/evm/types/types.go

```diff
@@ -76,6 +76,7 @@ type ChainCfg struct {
 	MinRequiredOutgoingConfirmations               null.Int
 	MinimumContractPayment                         *assets.Link
 	OCRObservationTimeout                          *models.Duration
+	LinkContractAddress                            null.String
```

Comment:
Can we sort these alphabetically while you're here

Reply:
Of course